foerder.html

```diff
@@ -314,21 +318,21 @@
 						<div id="personalInfo">
 							<div>
 								<div style="width: 36%;">
-									<input type="text" name="addrExtra" maxlength=30>
+									<input type="text" name="addrExtra" maxlength=30 pattern="[a-zA-Z0-9 *#\\?!:;'\x22=()\[\]<>.@,&_+-]*">
```

Comment:
I can input äüö.. and a qrcode is still build and there is no user visible error. I don't know about pattern in HTML inputs at all (yet).